docs/chart_template_guide/debugging.md

```diff
@@ -0,0 +1,9 @@
+# Debugging Templates
+
+Debugging templates can be tricky simply because the templates are rendered on the Tiller server, not the Helm client. And then the rendered templates are sent to the Kubernetes API server, which may reject the YAML files for reasons other than formatting.
+
+There are a few commands that can help you debug.
+
+- `helm lint` is your go-to tool for verifying that your chart follows best practices
+- `helm install --dry-run --debug`: We've seen this trick already. It's a great way to have the server render your templates, then return the resulting manifest file.
```

Comment:
Something I learned the other day, to solve the issue of YAML formatting issues when Helm tries to parse (which, if it fails to do, it won't output anything), you can comment out the template and run `--dry-run --debug`.